sys/crypto/doc.txt

```diff
@@ -25,6 +25,12 @@
  * While you can use the ciphers functions directly, you should resort to
  * the generic API for block ciphers whenever possible.
  *
+ * Additionally you need to set a CFLAG for each cipher you want to use in your Makefile:
+ *  * AES-128: CFLAGS += -DCRYPTO_AES
+ *  * 3DES:    CFLAGS += -DCRYPTO_THREEDES
+ *  * Twofish: CFLAGS += -DCRYPTO_TWOFISH
+ * Setting the CFLAGS initializes a sufficient large buffer size of the cipher_context_t used by the ciphers for en-/de-cryption operations
```

Comment:
That's perfect contentwise, but could you please add a linebreak and a fulstop at the end?